aspnetcore/fundamentals/websockets/samples/1.x/WebSocketsSample/EchoApp.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Diagnostics" Version="1.1.1" />
```

Comment:
These versions should go to the latest 1.1.x. Look them up at nuget.org. For example, this one should be `1.1.6`.

Reply:
Ok